cores/esp8266/Print.cpp

```diff
@@ -63,6 +63,53 @@ size_t Print::printf(const char *format, ...) {
     return len;
 }
 
+size_t Print::printf(const __FlashStringHelper * format, ...) {
```

Comment:
How about introducing Print:vprintf(PGM_P format, va_list argp) and make the other 2 use it?
```C++
size_t Print::printf(const __FlashStringHelper * format, ...) {
	va_list arg;
	va_start(arg, format);
	PGM_P fmt = reinterpret_cast<PGM_P>(format);
	size_t len = vprintf(fmt, arg);
	va_end(arg);
	return len;
```